README.md

```diff
@@ -74,6 +74,122 @@ Response headers:
     X-RateLimit-Reset: 1394506274
     X-RateLimit-Bucket: authorized-users
 
+## [Configuring Sphinx](./example.yaml)
+
+Rate limiting in Sphinx is managed by setting up `limits` in `yaml` configuration file. 
+Details about the configuration format can be found in the `[annotated example](./example.yaml)`
+
+It is important to understand the concept of `buckets` and `limits` to effectively configure a rate limiter.
+
+_Limit_: Limits define rate limiting policies that Sphinx enforces by counting requests in named _buckets_
```

Comment:
For consistency with bucket, make it singular: "A limit defines a rate limiting policy that Sphinx enforces by counting requests in named _buckets_."